### ISSUE TITLE
Using pre-trained model and do training on limited dataset

### ISSUE THREAD

**Issue by malpeddinilesh** [KaiyangZhou/deep-person-reid#422]
Hi @KaiyangZhou, 

First of all thank you so much for  your repo. 

Needed your guidance on using pre-trained model and do training on limited dataest.

I wanted to use OSNet-AIN model trained on MSMT17 combined all and then do transfer learning on our application specific dataset which is limited.

Based on your experience what are the best ways to get good result. I am thinking of following and I have questions also. 
1) Using two-stepped transfer learning: In this using max_epoch of 60 and fix base epoch of 10 while running engine.run
2) For Fine tuning pretrained model: small learning late for base layers and large learning late for classifer layers. Not sure only classifier layers tunning for my limited dataset would help. It needs to trained previous layers also.
3) Using only softmax or triplet+softmax will be helpful considering I have limited dataset.

It will be helpful, if you guide on using pre-trained model and training on limited dataset. 

Thank you for you help on this!

**Comment by KaiyangZhou** [comment]
Since your dataset is limited in size, I'd suggest using softmax first.

Both the transfer learning methods, i.e. 1 & 2, sound good to me. I would say try both to see which one works better.

**Comment by malpeddinilesh** [comment]
Thank you @KaiyangZhou for your quick response!

I am closing this.
